### DISCUSSION TITLE
Enviroment Variables - That Spans Collections

### DISCUSSION THREAD

**Discussion by RegencySoftware** [Kong/insomnia#5068] [Ideas] [answered]
Hi Everyone,

I am moving from Postman to use Insomnia.  

Like many people, I have multiple API collections.  In Postman I store some enviroment variables at the global level so that I can use these across collections or named environments and those can be used across collections.   I setup multiple collections in Insomnia but from what I see (base, sub, folder) only applies to a specific collection and there are no enviroment variables that span collections. 

Request: Feature where environments can be created and span collections.

**Comment by subnetmarco** [comment, marked as answer]
We [have announced](https://konghq.com/blog/product-releases/insomnia-9-3-ga) global environments where you can share environments across multiple collections.
